tests/nfs3-readdirplus/suricata.yaml

```diff
@@ -0,0 +1,14 @@
+%YAML 1.1
+---
+outputs:
+  - eve-log:
+      enabled: yes
+      filetype: regular
+      filename: eve.json
+      types:
+        - nfs
+
+app-layer:
+  protocols:
+    nfs:
+      enabled: yes
```

Comment:
Was the `suricata.yaml` added to enable `nfs` across all versions? Just to let you know, you could also achieve this by adding `--set app-layer.protocols.nfs.enabled=yes` in `test.yaml` `args` ;)

Reply:
I think that's a better option since there is really not much enabled for having a custom `suricata.yaml`
thanks for the suggestion!